### PR TITLE
feat(table): reusable table component for secrets/hooks/etc

### PR DESCRIPTION
**Pull request by davidvader**

### CODE REVIEW

**Comment by davidvader** [comment]
do not merge.... VSCode auto formatted that scss file. Fixing!

File: src/scss/_main.scss [hunk 1]
```diff
@@ -1313,6 +1345,34 @@ nav {
...
+  font-weight: 300;
+  margin-left: 1rem;
+}
+
+.field-header {
+  margin-top: 1rem;
+  margin-bottom: 0.25rem;
+}
+
+.-code {
```
Comment (wass3r) [inline src/scss/_main.scss:1372]: as a modifier, this should have a element it's attached to. I don't actually see this being used. If it was, would wrapping the content in `<code>` be sufficient?

File: src/elm/Table/Table.elm [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+customCell : Html msg -> Html.Attribute msg -> Html msg
+customCell element cls =
+    div [ class "cell", cls ]
+        [ span [] [ element ] ]
+
+
+{-| arrayCell : takes string array and renders cell
+-}
+arrayCell : List String -> String -> Html msg
+arrayCell images default =
```
Comment (wass3r) [inline src/elm/Table/Table.elm:128]: `images` seems weird here as a word for what looks to be textual content. what about `contents`?  and then rename the singular to `content`?

Comment (davidvader) [reply src/elm/Table/Table.elm:128]: doh! that was left in from working on the secrets table, my mistake

File: src/scss/_main.scss (hunk 1 above)
Comment (davidvader) [reply src/scss/_main.scss:1372]: agreed! ill update

Comment (davidvader) [reply src/scss/_main.scss:1372]: oh that must have slipped in while dev'ing, its not even used. good catch, thank you!

File: src/elm/Table/Table.elm (hunk 2 above)
Comment (davidvader) [reply src/elm/Table/Table.elm]: @jbrockopp dismissed your review to fix some copy pasta mistakes from ripping from my secrets branch

**Pull request merged**